flames.py

```diff
@@ -0,0 +1,47 @@
+n1 = str(input("Enter  your name: "))
+n2 = str(input("Enter your partner name: "))
+lst_n2 = list(n2)
+# print(lst_n2)
+l1 = len(n1)
+l2 = len(n2)
+full_len = l1+l2
+# print(full_len)
+for i in n1:
+    for j in lst_n2:
+        if i == j:
+            full_len = full_len-2
+            lst_n2.remove(j)
```

Comment:
not a good practice to change the list we are iterating!
May lead to errors which are difficult to debug.